docs/how-to/h-cluster-migration/h-cluster-replicaton.md

```diff
@@ -0,0 +1,165 @@
+
+Deploy mirrormaker:
+
+```bash
+juju deploy mirrormaker
+```
+
+After some time the app should show up in the model as blocked:
+
+```bash
+Unit              Workload  Agent  Machine  Public address  Ports           Message
+mirrormaker/0*    blocked   idle   4        10.86.75.16                     Integrator not ready to start, check if all relations are setup successfully
+```
+
+The MirrorMaker application has two endpoints that can be used with a Kafka cluster: `source` and `target`. The `source` endpoint is used to integrate with the active cluster, while the `target` endpoint is used to integrate with the passive cluster.
+
+```bash
+juju integrate kafka-connect mirrormaker
+juju integrate mirrormaker:source active
+juju integrate mirrormaker:target passive
+```
+
+After some time, the app should show up in the model as active:
+
+```bash
+Model  Controller  Cloud/Region         Version  SLA          Timestamp
+k      vms         localhost/localhost  3.6.3    unsupported  10:59:37+02:00
+
+App            Version  Status  Scale  Charm          Channel      Rev  Exposed  Message
+active         3.9.0    active      1  kafka          3/edge       205  no       
+kafka-connect           active      1  kafka-connect  latest/edge   20  no       
+mirrormaker             active      1  mirrormaker                   0  no       Task Status: UNASSIGNED
+passive        3.9.0    active      1  kafka          3/edge       205  no       
+
+Unit              Workload  Agent  Machine  Public address  Ports           Message
+active/0*         active    idle   0        10.86.75.171    9092,19092/tcp  
+kafka-connect/0*  active    idle   2        10.86.75.45     8083/tcp        
+mirrormaker/0*    active    idle   3        10.86.75.189    8080/tcp        Task Status: UNASSIGNED
+passive/0*        active    idle   1        10.86.75.153    9092,19092/tcp  
+```
+
+[note]
+Task status might show as UNASSIGNED since there are no replication tasks running yet. If the active Kafka cluster is idle, this is expected. The task status will change to `RUNNING` once the replication tasks are created and started.
+[\note]
+
+With this, the deployment is complete. The MirrorMaker charm will now start replicating data from the active cluster to the passive cluster.
+
+
+## Integrate active-active clusters.
+
+MirrorMaker allows for a deployment where both clusters are active. This means that data can be replicated from both clusters to each other. This is done by creating a MirrorMaker connector for each cluster. Two flows are needed in this scenario, one from cluster A to cluster B and one from cluster B to cluster A.
+
+In essence, it is equivalent to do two active-passive deployments, one on each direction. 
+
+It is also recommended to have two Kafka Connect deployments ready, one on each end of the replication.
+
+### Initial deployment
```

Comment:
note: That's the last heading of the guide. It feels like something is missing.

Reply:
Updated to just `Deployment`